thavalon-webapp/src/components/gameComponents/missionResults.tsx

```diff
@@ -1,5 +1,159 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { GameSocket, InboundMessage, InboundMessageType } from "../../utils/GameSocket";
+import { GameMessage, GameMessageType, MissionGoingMessage, MissionResultsMessage, Snapshot } from "./constants";
+import { OverlayTrigger, Tooltip } from "react-bootstrap";
+import "../../styles/gameStyles/missionResults.scss";
 
+enum MissionStatus {
+    Pending = "pending",
+    Going = "going",
+    Passed = "passed",
+    Failed = "failed",
+}
+
+interface MissionCardProps {
+    missionPlayers: String[],
+    missionStatus: MissionStatus,
+    missionNumber: number,
+    passes?: number,
+    fails?: number,
+    questing_beasts?: number,
+    reverses?: number,
+}
+
+/**
+ * MissionResults panel on game page, for showing mission results.
+ */
 export function MissionResults(): JSX.Element {
-    return <h1>Mission Results</h1>;
+    const [missionProps, setMissionProps] = useState<MissionCardProps[]>([1, 2, 3, 4, 5].map((x) => (
```

Comment:
What do you think of hiding missions that haven't gone yet instead of showing them as pending?

Reply:
will show what needs to be done for the mission instead, as per discussions. Will be done in another PR